engine/gamesys/wscript

```diff
@@ -26,23 +27,26 @@ def options(opt):
 def configure(conf):
     conf.load('waf_dynamo')
     conf.load('java')
+    target_os = conf.env['TARGET_OS']
 
     waf_ddf.configure(conf)
     waf_graphics.configure(conf)
     waf_physics.configure(conf)
     waf_render.configure(conf)
 
-    platform = conf.env['PLATFORM']
-
-    if platform in ("x86_64-macos",):
+    if target_os == TargetOS.MACOS:
```

Comment:
I guess it was bug here to add flags only for x86_64 macos variant.

Reply:
I consider setting all these configure steps in libraries' wscript deprecated, as it just duplicates things all over the scripts. I tend to prefer putting them in waf_dynamo.py instead.
Occasionally, it's beneficial to put config in the wscript, in order to make a test link the way one wants it to.